recipes/default.rb

```diff
@@ -24,18 +24,18 @@
 
 Chef::Resource.send(:include, Chocolatey::Helpers)
 
-install_ps1 = File.join(Chef::Config['file_cache_path'], 'install.ps1')
+install_ps1 = File.join(Chef::Config['file_cache_path'], 'chocolatey-install.ps1')
 
-template install_ps1 do
+remote_file install_ps1 do
   action :create
   backup false
-  source 'InstallChocolatey.ps1.erb'
-  variables :download_url => node['chocolatey']['install_vars']['chocolateyDownloadUrl']
+  source 'https://chocolatey.org/install.ps1'
+  not_if { chocolatey_installed? && (node['chocolatey']['upgrade'] == false) }
```

Comment:
Whoopsy, I didn't mean use the official one. There are a lot of folks who won't even want to reach out to the internet, so having a copy locally is good, even desired. I just meant to make it more up to date.